<ui version="4.0" >
 <class>AnimationControl</class>
 <widget class="QWidget" name="AnimationControl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>1</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="_outerFrame" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="_sliderSlider" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="_loopCheckBox" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If checked, the animation will loop continuously&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string>Loop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_stepBackwardButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="baseSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Take one step backward&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="iconSize" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_playBackwardButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Play backward&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_stopButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Stop playing&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_playForwardButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Play forward&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_stepForwardButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Take one step forward&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="iconSize" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Seconds / Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_speedSpinBox" >
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Set the number of seconds between animation steps&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="prefix" >
             <string/>
            </property>
            <property name="decimals" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <double>100.000000000000000</double>
            </property>
            <property name="minimum" >
             <double>0.100000000000000</double>
            </property>
            <property name="singleStep" >
             <double>0.100000000000000</double>
            </property>
            <property name="value" >
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
